src/components/application_manager/src/commands/hmi/navi_start_stream_request.cc

```diff
@@ -44,8 +44,10 @@ NaviStartStreamRequest::NaviStartStreamRequest(
     , EventObserver(application_manager.event_dispatcher())
     , retry_number_(0) {
   SDL_AUTO_TRACE();
+  const ApplicationManagerSettings& settings =
+      application_manager_.get_settings();
   std::pair<uint32_t, int32_t> stream_retry =
-      application_manager_.get_settings().start_stream_retry_amount();
+      settings.start_stream_retry_amount();
```

Comment:
@wolfylambova22, this variable used just once. I agree with @orhan-mehmedov, and so, see no reason why it should be here. Also, this change is not necessary and have no effect, nor for tests, which you trying to move, and especially, nor for realization.